actors/multisig/src/types.rs

```diff
@@ -18,8 +18,7 @@ use serde::{Deserialize, Serialize};
 pub const SIGNERS_MAX: usize = 256;
 
 /// Transaction ID type
-#[derive(Debug, Clone, Copy, Default, Serialize, Deserialize)]
-#[serde(transparent)]
+#[derive(Debug, Clone, Copy, Default, Serialize, Deserialize, Hash, Eq, PartialEq)]
```

Comment:
I think this is fine but calling out it changes this live type.

Reply:
Keep the `#[serde(transparent)]` part. Otherwise, the rest is fine if we need it (it just adds some code bloat).